src/containers/gamedata/index.js

```diff
@@ -68,72 +84,137 @@ export default function GameData(props) {
+      <Grid container spacing={3} direction="row" alignItems="center">
         <Grid item xs={5}>
+          <Typography align="center">HOME</Typography>
           <img
             src={getLogo(lineup.teams[1].data[0].abbreviation)}
             width="100px"
             style={{ display: "block", margin: "auto" }}
           />
           {prediction.data["Winning team"] ===
           lineup.teams[1].data[0].abbreviation ? (
-            <Typography align="center">WINNER</Typography>
+            <Typography align="center">PREDICTED WINNER</Typography>
           ) : (
-            <Typography align="center">LOSER</Typography>
+            <Typography align="center">PREDICTED LOSER</Typography>
           )}
         </Grid>
-        <Grid item xs={2}>
-          <Typography align="center">VS</Typography>
-        </Grid>
-
+            <Grid item xs={2}>
+                <Typography align="center">VS</Typography>
+            </Grid>
         <Grid item xs={5}>
+          <Typography align="center">AWAY</Typography>
           <img
             src={getLogo(lineup.teams[0].data[0].abbreviation)}
             width="100px"
             style={{ display: "block", margin: "auto" }}
           />
           {prediction.data["Winning team"] ===
           lineup.teams[0].data[0].abbreviation ? (
-            <Typography align="center">WINNER</Typography>
+            <Typography align="center">PREDICTED WINNER</Typography>
           ) : (
-            <Typography align="center">LOSER</Typography>
+            <Typography align="center">PREDICTED LOSER</Typography>
           )}
         </Grid>
       </Grid>
+      <Grid container className={classes.dateTime} xs={12} justify="center" alignItems="center">
+        <Grid item xs={12}>
+          <Typography align="center">{moment(lineup.game.startTime).format('LLL')} @ {lineup.references.venueReferences[0].name} in {lineup.references.venueReferences[0].city}</Typography>
+        </Grid>
+      </Grid>
+      </div>
     );
   }
 
-  if (prediction !== {}) {
+  // Check for lineup data and prediciton
+  if (lineup.teamLineups && prediction.data) {
+      // Gives an array of player objects with fielding position as their position
+      const awayFilteredFieldingLineup = lineup.teamLineups[0].expected.lineupPositions.filter(player => player.position.length < 3) || null;
+      const homeFilteredFieldingLineup = lineup.teamLineups[1].expected.lineupPositions.filter(player => player.position.length < 3) || null;
+       // Gives an array of player objects with batting order as their position
+      const awayFilteredBattingLineup = lineup.teamLineups[0].expected.lineupPositions.filter(player => player.position.length > 2) || null;
+      const homeFilteredBattingLineup = lineup.teamLineups[1].expected.lineupPositions.filter(player => player.position.length > 2) || null;
+       // Sorts lineup by fielding position
+       const awaySortedFieldingLineup = awayFilteredFieldingLineup.sort();
```

Comment:
Im not positive but i think you might be able to chain the .sort right onto the original .filter. So it would look like this.. `const awayFilteredFieldingLineup = lineup.teamLineups[0].expected.lineupPositions.filter(player => player.position.length < 3).sort || null;`  Might be interesting to see if that works. Ive done similar things like that before where i just chained multiple methods together.